src/background/make-chromium-happy.ts

```diff
@@ -1,15 +1,16 @@
 import {MessageType} from '../utils/message';
 import type {Message} from '../definitions';
+import {isChromium} from '../utils/platform';
 
 // This function exists to prevent Chrome from logging an error about
 // closed conduit. It just sends a dummy message in response to incomming message
 // to utilise open conduit. This response message is not even used on the other side.
 export function makeChromiumHappy() {
+    if (!isChromium) {
```

Comment:
This workaround was ever present in Chromium so it does not make much sense to do it on other platforms...